.github/workflows/unit_test.yml

```diff
@@ -0,0 +1,21 @@
+name: Unit Tests
+
+on:
+  push
```

Comment:
I verified that this action works... Thoughts on having it run on every push?

Reply:
I thought we were already running unit tests on every push to a pr - does this do something different?

Reply:
This repo currently doesn't do anything